### PR TITLE
feat(core): CHECKOUT-4909 Pass in attributes to stylesheet

### PR DESCRIPTION
**Pull request by vmparra**
## What?
Pass in attributes to stylesheet

## Why?
To be able to pass in attributes such as `integrity` and `crossorigin`

## Testing / Proof

- Unit

@bigcommerce/frontend @bigcommerce/checkout

### CODE REVIEW

File: src/stylesheet-loader.ts [hunk 1]
```diff
@@ -4,12 +4,17 @@ import BrowserSupport from './browser-support';
 
 export interface LoadStylesheetOptions {
     prepend: boolean;
+    attributes: StylesheetAttributes;
```
Comment (capsula4) [inline src/stylesheet-loader.ts:7]: should this be marked as optional?

Comment (vmparra) [reply src/stylesheet-loader.ts:7]: I believe this are mandatory, prepend won't serve of any purpose if we don't provide actual attributes.

Comment (capsula4) [reply src/stylesheet-loader.ts:7]: I think prepend is not related to the attributes itself, but rather, where it's loaded?

https://github.com/bigcommerce/script-loader-js/pull/11/files#diff-fad3242aa36c1c27d88b5b7c6a77e7f1R49

Comment (capsula4) [reply src/stylesheet-loader.ts:7]: I checked Script-loader#7 but I think it's probably a typescript bug. Should be marked as optional there too, as there is code that caters for the scenario when `attributes` is missing.

**Comment by vmparra** [comment]
@capsula4 I've addressed your comments ♻️

**Pull request merged**